packages/flutter/lib/src/material/material.dart

```diff
@@ -14,6 +14,35 @@ import 'theme.dart';
 /// Used by [InkHighlight] and [InkSplash], for example.
 typedef RectCallback = Rect Function();
 
+/// Interactive states that some of the Material component widgets can take on when
```

Comment:
You use `component` through this documentation, perhaps just using `widget` is better for Flutter readers.

Reply:
Yeah you are right that is probably more appropriate.